unit_tests/integration/test_nemesis_refresh.py

```
@@ -0,0 +1,73 @@
+
+    assert column_num
+    test_data = SstableLoadUtils.get_load_test_data_inventory(column_num, big_sstable=False, load_and_stream=False)
+
+    result = docker_scylla.run_nodetool(sub_cmd="cfstats", args="keyspace_refresh.standard1")
+
+    if result is not None and result.exit_status == 0:
+        key = "0x32373131364f334f3830"
+        # Check one special key before refresh, we will verify refresh by query in the end
+        # Note: we can't DELETE the key before refresh, otherwise the old sstable won't be loaded
+        #       TRUNCATE can be used the clean the table, but we can't do it for keyspace_refresh.standard1
+        query_verify = f"SELECT * FROM keyspace_refresh.standard1 WHERE key={key}"
```

Ruff (0.15.18)

[error] Possible SQL injection vector through string-based query construction

(S608)